http/handler_ccip.go

```diff
@@ -35,7 +35,7 @@ type CCIPResponse struct {
 
 // ExecCCIP handles GraphQL over Cross Chain Interoperability Protocol requests.
 func (c *ccipHandler) ExecCCIP(rw http.ResponseWriter, req *http.Request) {
-	store := req.Context().Value(storeContextKey).(client.Store)
+	db := req.Context().Value(dbContextKey).(client.DB)
```

Comment:
thought: There is nothing wrong with keeping the more restrictive type casting of `client.Store` since that is really what is needed here.